packages/datepicker/datepicker.scss

```diff
@@ -6,16 +6,18 @@
 $date-width: rem(46px);
 $calendar-padding: $component-spacing--xl;
 $calendar-width: ($date-width * 7) + ($calendar-padding * 2);
-$background-color: $snohvit;
-$popup-top-position: $label-height + $input-height + $bottom-padding + rem(2px);
+$background-color: white;
```

Comment:
not our color.

Reply:
Pure white _is_ a support color, but it needs to be implemented as a variable, along with the new red/blue support colors. I'll make a issue!